mllib/src/test/scala/org/apache/spark/mllib/clustering/LDASuite.scala

```diff
@@ -115,6 +115,15 @@ class LDASuite extends SparkFunSuite with MLlibTestSparkContext {
       assert(topicDistribution.toArray.sum ~== 1.0 absTol 1e-5)
     }
 
+    val top2TopicsPerDoc = model.topTopicsPerDocument(2).collect()
+    top2TopicsPerDoc.zip(topicDistributions).foreach {
```

Comment:
I've tested it many times and they are of the same order. Anyway, I can rewrite it using join if that looks better.